docs/SETTING_UP_INFRA.md

```diff
@@ -0,0 +1,66 @@
+# Setting up 3rd Party Infrastructure
+As of [KOGITO-4199](https://issues.redhat.com/browse/KOGITO-4199), 
+the operator no longer sets up 3rd party infrastructure as 
+to decouple the Kogito operator functionality from other 
+operators as well as allowing the user to setup the 3rd 
+party infrastructure more to their needs. As such, there is 
+now the onus on the user to setup the 3rd party 
+infrastructure, and this document is meant to help guide the 
+user to do so.
+
+# Infinispan
+## Installing Operator
+Currently, Kogito only supports running with Infinispan 11. 
```

Comment:
For release 1.8.0 we will support Infinispan 12.

Reply:
will support 1.8, or will only support 1.8+?

Reply:
I think we need to add @sutaakar Matrix in here. Maybe a link. The Compatibility Matrix should be updated every release and it's now in the main README file. Should be in a separate file or have a direct link to it (a session header will do).